test/spec/SearchResults/getFacetStats.js

```diff
@@ -37,43 +36,25 @@ var response = {
   }]
 };
 
-test('getFacetStats(facetName) returns stats for any facet or disjunctiveFacet', function(t) {
+test('getFacetStats(facetName) returns stats for any facet or disjunctiveFacet', function() {
   var searchParams = new SearchParameters({
     facets: ['age', 'country'],
     disjunctiveFacets: ['price']
   });
   var result = new SearchResults(searchParams, response.results);
 
-  t.throws(
-    bind(result.getFacetStats, result, 'city'),
-    Error,
-    'non defined facet should throw');
-  t.equal(
-    result.getFacetStats('country'),
-    undefined,
-    'should be undefined as "country" has no stats');
-  t.deepEqual(
-    result.getFacetStats('age'),
-    response.results[0].facets_stats.age,
-    'should return the same stats data as in the response');
-  t.deepEqual(
-    result.getFacetStats('price'),
-    response.results[0].facets_stats.price,
-    'should return the same stats data as in the response');
-
-  t.end();
+  expect(bind(result.getFacetStats, result, 'city')).toThrowError(Error);
+  expect(result.getFacetStats('country')).toBe(undefined);
```

Comment:
We can use [`toBeUndefined()`](https://jestjs.io/docs/en/expect#tobeundefined).

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453